fabric8/fabric8.addon

```diff
@@ -0,0 +1,37 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+# Required-Vars: GITHUB_OAUTH_CLIENT_ID,GITHUB_OAUTH_CLIENT_SECRET
+
+echo NOTE: To delete this addon, execute:
+echo   oc delete project/fabric8 project/developer oauthclient/fabric8-online-platform
+
+echo "Enabling CORS"
+openshift ex config patch /var/lib/origin/openshift.local.config/master/master-config.yaml --patch='{"corsAllowedOrigins": [".*"]}'
+docker restart origin
+
+# TODO how best to wait for openshift to be back?
+sleep 10
+
+oc adm new-project developer
+oc adm new-project fabric8
+
+echo Installing the Fabric8 System using GitHub Client ID #{GITHUB_OAUTH_CLIENT_SECRET}
+oc new-app --param APISERVER_HOSTPORT="#{ip}:8443" --param GITHUB_OAUTH_CLIENT_SECRET=#{GITHUB_OAUTH_CLIENT_SECRET} --param GITHUB_OAUTH_CLIENT_ID=#{GITHUB_OAUTH_CLIENT_ID} https://raw.githubusercontent.com/fabric8io/fabric8-resources/master/fabric8-system-openshift.yml -n fabric8
```

Comment:
the reason we do this is so that the fabric8 YAML can move much more quickly than the fabric8-addon. we've yet to have 1 commit after weeks so far - while we tend to do 1-2 releases per day of the fabric8 YAML